src/main/java/com/dev/cinema/model/dto/OrderDtoResponse.java

```diff
@@ -0,0 +1,42 @@
+package com.dev.cinema.model.dto;
+
+import java.util.List;
+
+public class OrderDtoResponse {
+    private Long id;
+    private List<Long> ticketsIds;
+    private String orderDate;
```

Comment:
```suggestion
    private LocalDateTime orderDate;
```

Reply:
waiting for fix

Reply:
Fixed, but could you send me some link or short message why LocalDateTime?

Reply:
It's better to send a LocalDateTime value. It will be parsed to String automatically. Otherwise, you can use your custom LocalDateTime deserializer if it is required.